tox.ini

```diff
@@ -12,10 +12,9 @@ isolated_build = true
 # Pass through the following environemnt variables which are needed for the CI
 passenv = HOME WINDIR CC CI
 
-# FIXME: Cannot do this, need to force C-ext to build below.
 # Run the tests in a temporary directory to make sure that we don't import
 # package from the source tree
-#changedir = .tmp/{envname}
+changedir = {homedir}
```

Comment:
Does it have to be `homedir`? Did `changedir = .tmp/{envname}` not work?

Reply:
Does it matter?

Reply:
Yes, `homedir` might be pre-populated with stuff. `.tmp/{envname}` is clean.

Reply:
The point is that tests do run in the install directory unlike what it was before - the source directory. Whether they run in
```
home/.tox/py39.../lib/python3.9/site-packages/synphot
```
or 
```
chdir .tmp/py39env
../../home/.tox/py39env/lib/python3.9/site-packages/synphot
```
does it make a difference?

Reply:
I'd rather you change it to `.tmp/{envname}` anyway, if either one works. That is what `astropy` is using and easier for me to copy paste from upstream in the future.

Reply:
Done